src/Microsoft.Health.Dicom.Api/Modules/SecurityModule.cs

```diff
@@ -25,11 +28,14 @@ namespace Microsoft.Health.Dicom.Api.Modules;
 public class SecurityModule : IStartupModule
 {
     private readonly SecurityConfiguration _securityConfiguration;
+    private readonly Dictionary<string, string> _audienceToSchemeMapper;
+    private string _defaultSchemeName;
 
     public SecurityModule(DicomServerConfiguration dicomServerConfiguration)
     {
         EnsureArg.IsNotNull(dicomServerConfiguration, nameof(dicomServerConfiguration));
         _securityConfiguration = dicomServerConfiguration.Security;
+        _audienceToSchemeMapper = new Dictionary<string, string>();
```

Comment:
Would authority to schema mapper make more sense? In the case of AAD, you could have a multi-tenant application that represents your service in multiple tenants, but they would have different authorities.

Reply:
Also, I believe that there's on a single `iss` claim in a `jwt` so that may also simplify other logic below.

Reply:
I was initially thinking of doing this but noticed that the authority and issuer were a bit different - https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/ vs https://sts.windows.net/72f988bf-86f1-41af-91ab-2d7cd011db47/
But I guess it could make sense to get the tenant Id out of these claims and match on that. I've updated to do that